src/library-authoring/data/apiHooks.ts

```diff
@@ -542,3 +544,11 @@ export const useUpdateComponentCollections = (libraryId: string, usageKey: strin
     },
   });
 };
+
+export const useComponentDownstreamContexts = (usageKey: string) => (
```

Comment:
@rpenido Docstring is missing here

Reply:
Thanks! Fixed here:  aadf958905dc654a97890910dd5ce773c7fe3748